quilt/tools/command.py

```diff
@@ -148,6 +148,12 @@ def _parse_package(name):
             raise ValueError
     except ValueError:
         raise CommandException("Specify package as owner/package_name.")
+
+    try:
+        PackageStore.check_name(owner, pkg)
+    except StoreException as ex:
+        raise CommandException(str(ex))
```

Comment:
Ok, I'll do that in a separate PR. "Rename package" wouldn't be appropriate if e.g. you're running `quilt install ...`; in that case, you just got the name wrong, and there's nothing to rename.